src/Symfony/Component/Security/Http/Authenticator/Passport/Badge/UserBadge.php

```diff
@@ -27,6 +28,8 @@
  */
 class UserBadge implements BadgeInterface
 {
+    public const MAX_USERNAME_LENGTH = 4096;
```

Comment:
Perhaps out of context but why reintroducing username here? Why not user identifier?